.gitignore

```diff
@@ -10,3 +10,8 @@ __pycache__/
 .mypy_cache
 *.egg-info/
 */*.rock
+*/*.snap
+# charmcraft-related directories
+prime/
```

Comment:
When are these created?

Reply:
I believe this was because I used a custom version of charmcraft, those directory should not appear with the stable versions. But I think we can still keep them here in case someone runs charmcraft in destructive mode

Reply:
thx for the explanation